specification/resources/resource-manager/Microsoft.Solutions/stable/2017-09-01/managedapplications.json

```diff
@@ -1357,6 +1374,49 @@
           "type": "string"
         }
       }
+    },
+    "OperationDisplay": {
+      "properties": {
+        "provider": {
+          "type": "string",
+          "description": "Operation provider."
+        },
+        "resource": {
+          "type": "string",
+          "description": "Operation resource."
+        },
+        "operation": {
+          "type": "string",
+          "description": "Operation."
+        },
+        "description": {
+          "type": "string",
+          "description": "Operation description."
+        }
+      }
```

Comment:
Please mark all properties are readonly given user cannot update any of these

"readOnly": true,